scripts/windows-download-cache-and-build-module-wheels.ps1

```diff
@@ -11,6 +11,14 @@
 #
 #     > windows-download-cache-and-build-module-wheels.ps1 11
 #
+# - other parameters are passed to setup.py. If one of the parameters is "--",
+#   the following parameters will be passed to cmake.
+#     For instance, for Python 3.11, excluding nvcuda.dll during packaging
+#     and setting RTK_USE_CUDA ON during configuration:
+#
+#     > windows-download-cache-and-build-module-wheels.ps1 11 --exclude-libs nvcuda.dll "--" -DRTK_USE_CUDA:BOOL=ON
```

Comment:
I also pass options to Linux [here](https://github.com/RTKConsortium/RTK/pull/526/files#diff-b605ee46fbd21ee607667ab55c9a27d46cc6981d9468c2db4bed457d80aa9ac3R51) but the mechanism is not the same: https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh#L40. This `--` approach (on Windows) is not optimal because the double quotes are mandatory for `windows-download-cache-and-build-module-wheels.ps1`, it seems to be lost otherwise... I am not sure why the `-c` option has not been used for Windows, I'm sure @LucasGandel will know.

Reply:
The `-c` option does not exists in Windows scripts, which is why it has not been used (see initial commit [here](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/commit/54677270f1a1eb707bfc234771f16836de526c9a)). I agree with @tbirdso that we should add such flag for consistency.
The `--` argument just tells argparse to treat all trailing arguments as a single positional argument. This is mandatory in this case because CMake arguments start with `-` and would be wrongly interpreted by argparse otherwise

Reply:
Added https://github.com/InsightSoftwareConsortium/ITKPythonPackage/issues/250 to track.